.prettierignore

```diff
@@ -0,0 +1,2 @@
+cypress/tests/**/*
+lib/pkp/cypress/**/*
```

Comment:
This we can remove in future, its just to avoid big formatting changes for this PR.